src/utilities/effects.ts

```diff
@@ -7184,7 +7184,7 @@ export function replacePaths(
   }
   const result: ArgumentsMap = {};
   for (const parameterName in modelParameters) {
-    const parameter: Parameter = modelParameters[parameterName];
+    const parameter: Parameter = modelParameters[parameterName] as Parameter;
```

Comment:
I think we can remove this `as Parameter`now  & it will work correctly with the generics